renderdoc/driver/vulkan/vk_hookset_defs.h

```diff
@@ -677,7 +678,8 @@
   CheckExt(EXT_mesh_shader, VKXX);                            \
   CheckExt(EXT_scalar_block_layout, VK12);                    \
   CheckExt(KHR_deferred_host_operations, VKXX);               \
-  CheckExt(KHR_acceleration_structure, VKXX);
+  CheckExt(KHR_acceleration_structure, VKXX);                 \
+  CheckExt(KHR_ray_query, VKXX);
```

Comment:
FYI, it's harmless to add here but strictly speaking we only need these extension enabled checks to gate functionality - like an entry point, or if we need to check during capture/replay if an extension is enabled. Anything that's shader only like this doesn't have to be included.